frontend/app/src/components/settings/api-keys/ExchangeKeysForm.vue

```diff
@@ -281,6 +267,59 @@ onMounted(() => {
 
 const krakenAccountTypes = KrakenAccountType.options;
 const exchanges = SUPPORTED_EXCHANGES;
+
+const sensitiveFieldEditable = computed(() => !get(edit) || get(editKeys));
+
+const rules = {
+  name: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.name.non_empty'),
+      requiredIf(computed(() => !get(edit)))
+    )
+  },
+  newName: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.name.non_empty'),
+      requiredIf(edit)
+    )
+  },
+  apiKey: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.api_key.non_empty'),
+      requiredIf(sensitiveFieldEditable)
+    )
+  },
+  apiSecret: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.api_secret.non_empty'),
+      requiredIf(sensitiveFieldEditable)
+    )
+  },
+  passphrase: {
+    required: helpers.withMessage(
+      tc('exchange_keys_form.passphrase.non_empty'),
+      requiredIf(
+        computed(() => get(sensitiveFieldEditable) && get(requiresPassphrase))
+      )
+    )
+  }
+};
+
+const v$ = useVuelidate(
+  rules,
+  {
+    name: computed(() => get(exchange).name),
+    newName: computed(() => get(exchange).newName),
+    apiKey: computed(() => get(exchange).apiKey),
+    apiSecret: computed(() => get(exchange).apiSecret),
+    passphrase: computed(() => get(exchange).passphrase)
```

Comment:
This is as the one before, can you see if we can use 
either `reactivePick` or `reactiveOmit`
https://vueuse.org/shared/reactiveomit/#reactiveomit

pick should be preferred if the omitted elements are more than the picked,
and omit if the picked elements are more than the omitted.